seeker/views.py

```diff
@@ -85,7 +86,11 @@ def header(self):
             q['s'] = self.field
         next_sort = 'descending' if sort == 'Ascending' else 'ascending'
         sr_label = (' <span class="sr-only">(%s)</span>' % sort) if sort else ''
-        html = '<th class="%s"><a href="?%s" title="Click to sort %s" data-sort="%s">%s%s</a></th>' % (cls, q.urlencode(), next_sort, q['s'], self.header_html, sr_label)
+        if self.field_definition:
+            span = '<span title="%s" class ="fa fa-question-circle"></span>'.format(self.field_definition)
```

Comment:
The %s doesn't work with .format().  This needs to be a {}

```suggestion
            span = '<span title="{}" class ="fa fa-question-circle"></span>'.format(self.field_definition)
```